source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of the new system can get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We have build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficient operating system to run Docker like a hypervisor. It has installed just enough software to run a supervisor. We have also focussed on security, there are, for example, no default passwords and we use [AppArmor] to protect the applications and containers on HassOS.
+
+Key features:
+
+- Safe and secure updating with [Rauc] over USB or internet (OTA)
+- Uses an LZ4 compressed root filesystem and parts of the memory
+- Read-only root filesystem, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT Linux kernel
+- Latest Docker-ce version
+- Fully supported NetworkManager
+- Bluetooth support using Bluez
+- Supports lots different Hardware
+
+## {% linkable_title Migration %}
+
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old (ResinOS) too new system (HassOS).
+
+You need to perform the following steps to upgrade:
+
+1. If you have installed the Bluetooth add-on, please remove it, since it is not required anymore.
```

Comment:
What happens if they keep it enabled, will the snapshot be able to be restored?

Reply:
The end of the world?

No sure, I'll guess it simply doesn't work, but still will boot. So removing it afterward?
@pvizeli Could you confirm?